src/controllers/commands/CommandController.cpp

```diff
@@ -3211,6 +3212,8 @@ void CommandController::initialize(Settings &, Paths &paths)
 
     this->registerCommand("/shield", &commands::shieldModeOn);
     this->registerCommand("/shieldoff", &commands::shieldModeOff);
+
+    this->registerCommand("/c2:loggingrules", &commands::setLoggingRules);
```

Comment:
Naming is the hardest part - I think I would prefer `/c2-set-logging-rules` but happy to discuss it further before merging this in

Reply:
Sounds like a good name to me.